packages/miniflare/test/plugins/core/modules.spec.ts

```diff
@@ -217,16 +200,18 @@ test("Miniflare: cannot automatically collect modules from dynamic import expres
 		scriptPath,
 	});
 
-	let error = await t.throwsAsync(mf.ready, {
-		instanceOf: MiniflareCoreError,
-		code: "ERR_MODULE_DYNAMIC_SPEC",
-	});
+	let error: Error | undefined;
+	try {
+		await mf.ready;
+	} catch (e) {
+		error = e as Error;
+	}
+	expect(error).toBeDefined();
```

Comment:
Not needed as we have an assertion on the next line (which also narrows the type)

Reply:
And similarly throughout this file